lib/one_wire/ibutton/protocols/dallas/protocol_ds1971.c

```diff
@@ -0,0 +1,258 @@
+#include "protocol_ds1971.h"
+
+#include <core/core_defines.h>
+#include <toolbox/pretty_format.h>
+
+#include "dallas_common.h"
+
+#define DS1971_FAMILY_CODE 0x14U
+#define DS1971_FAMILY_NAME "DS1971"
+
+#define DS1971_SRAM_DATA_SIZE 32U
+#define DS1971_SRAM_PAGE_SIZE 32U
+#define DS1971_COPY_SCRATCH_TIMEOUT_US 250U
+
+#define DS1971_DATA_BYTE_COUNT 4U
+
+#define DS1971_SRAM_DATA_KEY "Sram Data"
```

Comment:
According to the [iButton file format docs](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/file_formats/iButtonFileFormat.md), protocols can specify their own custom data fields.
Since DS1971 is a EEPROM device, this key should be "Eeprom Data" to reflect that. This also should be added to the aforementioned documentation.